nrf_modem/doc/sockets/pdn.rst

```diff
@@ -0,0 +1,61 @@
+.. _packet_data_networks:
+
+Packet Data Networks (PDN)
+##########################
+
+.. contents::
+   :local:
+   :depth: 1
+
+The Modem library supports selecting which Packet Data Network (PDN) to use on a network socket and for DNS queries.
+The configuration of Packet Data Protocol (PDP) contexts, and the activation of PDN connections are not handled by the Modem library.
+To configure PDP contexts and activate PDN connections, the application must use the packet domain AT commands.
+
+When performing network operations on any PDN, the application ensures that the PDN connection is available.
+For more information about how to configure PDP contexts, activate PDN connections, and determine their state, see the `Packet domain commands`_ section in the nRF9160 AT Commands Reference Guide or the same section in the `nRF91x1 AT Commands Reference Guide`_ depending on the SiP you are using.
+
+The packet domain AT commands can be used to manage packet data networks.
```

Comment:
I think this sentence is redundant with the one above, we can remove it.

Reply:
Removed.